bench/utils.py

```diff
@@ -150,7 +151,13 @@ def init(path, apps_path=None, no_procfile=False, no_backups=False,
 
 	setup_env(bench_path=path, python=python)
 
-	make_config(path)
+	extra_config = None
+	if clone_full:
+		extra_config = {
+			'shallow_clone': False
+		}
+
+	make_config(path, extra_config)
```

Comment:
You could pass the clone request as a keyword argument and receive it at `make_config` as `kwargs` instead.

```suggestion
	make_config(path, shallow_clone=not clone_full)
```

Reply:
Thank you for the suggestion, added a patch with the suggested change.